.github/workflows/pr-closed.yml

```diff
@@ -7,20 +7,7 @@ on:
 
 jobs:
   docs:
-    name: Remove Docs Preview
+    name: Trigger
```

Comment:
Why does this still exist as a separate workflow?

Reply:
Because I need something to happen when the PR is closed

Reply:
But why can't `docs-remove-preview` not be triggered by the PR closing directly?

Reply:
Because then you don't have access to repo secrets. Unless I'm missing something and it's possible to trigger a workflow with the repo secrets directly from a PR.

Reply:
Ahh, I see 👍🏼